source/iostreams.tex

```diff
@@ -13011,7 +13011,7 @@
 
 \begin{ncbnf}
 \nontermdef{filename}\br
-    \textnormal{non-empty sequence of characters other than \grammarterm{directory-separator} characters}
+    \textnormal{non-empty sequence of characters other than directory separator characters}
```

Comment:
A *directory-separator* is a sequence of *preferred-separator*s and *fallback-separator*s. A "directory separator character" should probably be an element of a *directory-separator*, i. e. a *preferred-separator* or a *fallback-separator*.

Would adding a definition of "directory separator character" be editorial? If "directory separator character" is defined, should the definition of *preferred-separator* be changed to not mention "directory separator character", in order to avoid seemingly circular definition?

https://github.com/cplusplus/draft/blob/88e574edc674e85911ea1a5b4e87c07457d97bf1/source/iostreams.tex#L13421-L13424